doc/s9s.dbversions.1

```diff
@@ -0,0 +1,104 @@
+.TH S9S-DBVERSIONS 1 "December 19, 2023"
+
+.SH NAME
+s9s-dbversions \- Command-line client for the Severalnines ClusterControl server.
+
+.SH SYNOPSIS
+.B s9s-dbversions
+.RI [OPTION]... 
+.RI [PATH]...
+
+.SH DESCRIPTION
+\fBs9s-dbversions\fP is a command-line tool for ClusterControl, which can be used to 
+obtain database versions of the different clusters supported by ClusterControl.
+
+.SH OPTIONS
+.SS "Main Option"
+The application should always be started using a main option that sets what operation
+should be performed. This "main option" should be one of the following:
+
+.TP
+.B --cluster-types
+Gets the list of supported cluster types.
+Returned values can be used as an argument to the --cluster-type option (upper or lower case).
+
+.B EXAMPLE
+.nf
+s9s dbversions --cluster-types                                                                                                                                                                                                                                             CLUS-3235 ✗
+MYSQLCLUSTER
+REPLICATION
+MYSQL_SINGLE
+MONGODB
+POSTGRESQL_SINGLE
+GROUP_REPL
+REDIS
+MSSQL_SINGLE
+MSSQL_AO_ASYNC
+MSSQL_AO_SYNC
+ELASTIC
+GALERA
+.fi
+
+.TP
+.B --vendors
+Gets the list of supported vendors on all supported clusters.
+If no version is retrieved for a specific cluster type-vendor, it means that it is not supported.
+
+
+.B EXAMPLE
+.nf
+s9s dbversions --vendors                                                                                                                                                                                                                                                   CLUS-3235 ✗
+percona
+mariadb
+oracle
+codership
+10gen
+mongodbenterprise
+microsoft
+elasticsearch
+redis
+postgresql
+enterprisedb
+.fi
+
+.TP
+.B --list-versions
+Lists the available 1 or 2 digits versions supported by ClusterControl for a given 
+cluster type and vendor.
+
+.TP
+.B --list-3d-versions
+Lists the available 3 digits versions supported by ClusterControl for a given 
+cluster type and vendor (if supported). If not, a bad request will be returned.
+
+.B EXAMPLE
+.nf
+s9s dbversions --list-versions --cluster-type galera --vendor mariadb
```

Comment:
Now I get `Invalid operation 'getDbVersions' on '/v2/info'.`:
```
19:42:41 s9s-tools$ ./s9s/s9s dbversions --list-versions --cluster-type galera --vendor mariadb
Invalid operation 'getDbVersions' on '/v2/info'.
19:42:50 s9s-tools$ gitstbr
On branch CLUS-3235
Your branch is up to date with 'origin/CLUS-3235'.

nothing to commit, working tree clean
---------------

* 05dbc053 - (3 hours ago) CLUS-3235 remove unneded iostream - alvaro-vinuela (HEAD -> CLUS-3235, origin/CLUS-3235)
* 14b92b67 - (3 hours ago) CLUS-3235 speeling check on man page and other PR comments - alvaro-vinuela
* 6e477a60 - (28 hours ago) CLUS-3235 adding cmon-user and pass options to 'dbversions' subcommand - alvaro-vinuela
* 3296852e - (29 hours ago) CLUS-3235 adding 'dbversions' subcommand - alvaro-vinuela
```
Did you test the command? Does it work for you?

Reply:
I probably need `cmon` with the related changes to get the command working

Reply:
Works:
```
23:21:31 ~$ s9s dbversions --list-versions --cluster-type galera --vendor mariadb
10.4
10.5
10.6
10.7
10.8
10.9
10.10
10.11
23:22:00 ~$ 
```
:)

Reply:
Will continue review tomorrow...

Reply:
FYI:
```
23:23:48 ~$ s9s dbversions --list-3d-versions --cluster-type galera --vendor mariadb
10.4.32
10.5.23
10.6.16
10.10.7
10.11.6
11.0.4
11.1.3
11.2.2
11.3.1
23:24:00 ~$ s9s dbversions --list-3d-versions --cluster-type mongodb --vendor 10gen
Invalid operation 'getDb3dVersions' on '/v2/dbversions'.
23:24:14 ~$ s9s dbversions --list-versions --cluster-type mongodb --vendor 10gen
4.4
5.0
6.0
23:24:24 ~$ 
```